### PR TITLE
Enable includecheck leg

### PR DESCRIPTION
**Pull request by dcristoloveanu**

### CODE REVIEW

File: src/ctest.c [hunk 1]
```diff
@@ -232,7 +235,7 @@ size_t RunTests(const TEST_FUNCTION_DATA* testListHead, const char* testSuiteNam
             /*revert console to initial state*/
             if (!SetConsoleMode(std_out_handle, console_mode_initial))
             {
-                LogWarning("Error resetting console mode to initial value of %" PRIx32 ". GetLastError()=%" PRIx32 "", console_mode_initial, GetLastError());
```
Comment (mattdurak) [inline src/ctest.c:235]: >PRIx32 [](start = 80, length = 6)

probably still PRIx32 (like above)

**Comment by mattdurak** [comment]
>       runTestsInIsolation: true

Does this actually run? Or do we not generate the DLL's for vstest? Asking to see if the tests build as c++ sometimes

---
Refers to: build/devops_gated.yml:119 in 0bf24d2. [](commit_id = 0bf24d221e89fa3d1a071ae0c5fc651b707dc1cd, deletion_comment = False)

**Comment by dcristoloveanu** [comment]
>       runTestsInIsolation: true

"##[warning]No test sources found matching the given filter '**\*test*.dll,!**\obj\**'


---
In reply to: [764073183](https://github.com/Azure/ctest/pull/74#issuecomment-764073183) [](ancestors = 764073183)

---
Refers to: build/devops_gated.yml:119 in 0bf24d2. [](commit_id = 0bf24d221e89fa3d1a071ae0c5fc651b707dc1cd, deletion_comment = False)

**Comment by dcristoloveanu** [comment]
>       runTestsInIsolation: true

There is a separate task to fix this, but I'll fix it now anyway

---
In reply to: [764075247](https://github.com/Azure/ctest/pull/74#issuecomment-764075247) [](ancestors = 764075247,764073183)

---
Refers to: build/devops_gated.yml:119 in 0bf24d2. [](commit_id = 0bf24d221e89fa3d1a071ae0c5fc651b707dc1cd, deletion_comment = False)

**Comment by dcristoloveanu** [comment]
>       runTestsInIsolation: true

Well, it turns out that this is the repo of the C test runner. So why should we run anything with *cppunitest*?
So I removed the task.

---
In reply to: [764076188](https://github.com/Azure/ctest/pull/74#issuecomment-764076188) [](ancestors = 764076188,764075247,764073183)

---
Refers to: build/devops_gated.yml:119 in 0bf24d2. [](commit_id = 0bf24d221e89fa3d1a071ae0c5fc651b707dc1cd, deletion_comment = False)

File: src/ctest.c (hunk 1 above)
Comment (dcristoloveanu) [reply src/ctest.c:235]: Fixed.

---
In reply to: [561430305](https://github.com/Azure/ctest/pull/74#discussion_r561430305) [](ancestors = 561430305)

**Pull request merged**